charts/monitoring/prometheus/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: prometheus
-version: v9.9.9-dev
+version: v9.9.10-dev
```

Comment:
this doesn't have to be changed anymore, please leave it at 9.9.9 :)

Reply:
Done. Thanks for noticing it.